app/src/main/java/com/framgia/englishconversation/screen/imagedetail/ImageDetailViewModel.java

```diff
@@ -54,11 +57,17 @@ public void onItemUserNameClick(TimelineModel item) {
                     mContext.startActivity(
                             ProfileUserActivity.getInstance(mContext, item.getCreatedUser()));
                 }
+
+                @Override
+                public boolean onItemLongClick(View viewGroup, TimelineModel item) {
+                    return false;
+                }
```

Comment:
ERROR: 'method def rcurly' have incorrect indentation level 16, expected level should be one of the following: 8, 12.